### ISSUE TITLE
usb: Error: BUG: _current is zero!

### ISSUE THREAD

**Issue by alex-ab** [genodelabs/genode#2153]
```
[init -> vbox] Error: device already claimed
[init -> usb_drv] Warning: __wait_event():dd empty list in wq: 0x1175f20
[init -> usb_drv] Error: BUG: _current is zero!
```
This bug has been seen during detachment of a raw usb device (having a usb raw driver) and re-attaching it. Apparently it is a bug in the configuration of the integrator, nevertheless the driver should not go out of service.

**Comment by cnuke** [comment]
I tried to reproduce the problem with a driver using the Usb session directly but without any success. In this case claiming a device multiple times does not lead to the observed page fault. I will try to reproduce the issue with vbox.

**Comment by alex-ab** [comment]
Fixed by #2466.
